frontend/src/components/FifthExamComponent/FifthExamFormComponent.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import './FifthExamFormComponent.css';
+import {
+  Paper, TextField, FormControl, InputLabel, Select, MenuItem, Button, FormLabel, RadioGroup,
+  FormControlLabel, Radio,
+} from '@material-ui/core';
+import CreatePDFButton from '../Buttons/CreatePDFButton';
+
+const referenzfachMock = [{ id: 1, name: 'Mathe' }, { id: 2, name: 'Deutsch' }, { id: 3, name: 'English' }];
+const examinerMock = [{ id: 1, name: 'Mr. Jonson' }, { id: 2, name: 'Mrs. Heathrow' }, { id: 3, name: 'Miss Daisy' }];
+const bezugsfachMock = [{ id: 1, name: 'Geschichte' }, { id: 2, name: 'Deutsch' }, { id: 3, name: 'Geografie' }];
+
+function FifthExamFormComponent() {
+  const aDate = new Date();
+  const dueDate = `${aDate.getDate()}.${aDate.getMonth() + 1}.${aDate.getFullYear()}`;
+
+  const [examType, setExamType] = useState('');
+  const [studentName, setStudentName] = useState('');
+  const [studentPartner, setStudentPartner] = useState('');
+  const [chosenTopicArea, setChosenTopicArea] = useState('');
+  const [referenzfach, setReferenzfach] = useState('');
+  const [examiner, setExaminer] = useState('');
+  const [bezugsfach, setBezugsfach] = useState('');
+
+  const referenzfachList = referenzfachMock;
+  const examinerList = examinerMock;
+  const bezugsfachList = bezugsfachMock;
+
+  const handleExamTypeInputChange = (event) => {
+    setExamType(event.target.value);
+  };
+  const handleStudentNameInputChange = (event) => {
+    setStudentName(event.target.value);
+  };
+  const handleStudentPartnerInputChange = (event) => {
+    setStudentPartner(event.target.value);
+  };
+  const handleChosenTopicAreaInputChange = (event) => {
+    setChosenTopicArea(event.target.value);
+  };
+  const handleReferenzfachInputChange = (event) => {
+    setReferenzfach(event.target.value);
+  };
+  const handleExaminerInputChange = (event) => {
+    setExaminer(event.target.value);
+  };
+  const handleBezugsfachInputChange = (event) => {
+    setBezugsfach(event.target.value);
+  };
+
+  const handleGeneratePDF = () => {
+    alert('pdf generiert...');
+    const pdf = {
+      examType,
+      studentName,
+      studentPartner,
+      chosenTopicArea,
+      referenzfach,
+      examiner,
+      bezugsfach,
+    };
+    console.log(pdf.toString());
+  };
+
+  const handleSubmitFifthExamForm = () => {};
+
+  return (
+    <div style={{ overflow: 'scroll' }}>
+      <Paper
+        className="fifthExamPaper"
+        style={{
+          display: 'flex', flexDirection: 'column', width: '70%', marginRight: '3vw', padding: '5vw', margin: 'auto',
+        }}
+      >
+        <h1 id="headingFifthExaam">Pruefungskomponente</h1>
```

Comment:
wie wir jetzt in der Telegramgruppe besprochen haben pack die heading außerhalb des Papers. Dann hast du auch etwas platz zwischen der toolbar und dem paper.

Außerdem kannst du hier ruhig umlaute verwenden. JavaScript bzw. TypeScript sollte damit umgehen können :D